Intersect.Client/MonoGame/IntersectGame.cs

```diff
@@ -117,19 +114,29 @@ private IntersectGame(IClientContext context, Action postStartupAction)
             Interface.Interface.GwenInput = new IntersectInput();
             Controls.Init();
 
-            Window.Position = new Microsoft.Xna.Framework.Point(-20, -2000);
+            // Reuse Point object instead of creating a new one each time.
+            var hiddenWindowPosition = new Microsoft.Xna.Framework.Point(-20, -2000);
+            Window.Position = hiddenWindowPosition;
```

Comment:
1. Point is not an object
2. you are not reusing it anywhere

Change this back